### ISSUE TITLE
DECISION_PATHS_CAVEATS - Question

### ISSUE THREAD

**Issue by FelixNeutatz** [TeamHG-Memex/eli5#247]
Hi,

thank you, for this great library. I have a question about the first decision paths caveat. The first caveat says (https://github.com/TeamHG-Memex/eli5/blob/master/eli5/_decision_path.py#L24):

> Feature weights just show if the feature contributed positively or negatively to the final score, and **does show** how increasing or decreasing the feature value will change the prediction.

How is this a caveat, if the feature weights **do show** how increasing or decreasing the feature value will change the prediction? Or is there a 'not' missing?

If the 'not' is not missing, can somebody explain to me how the feature weights show how increasing or decreasing the feature value will change the prediction?

Thank you for your great work,
Felix

**Comment by lopuhin** [comment]
@FelixNeutatz there is "not" missing I believe, thanks for the catch!

**Comment by lopuhin** [comment]
I'll reopen that to make sure we fix the caveat :)

**Comment by FelixNeutatz** [comment]
ok thanks again :)

**Comment by lopuhin** [comment]
FTR, the build after  eeb3735 was initially broken due to failure to connect to PyPI in one of the builds during setuptools update at the start. Restarting the build fixed it.
